runelite-client/src/main/java/net/runelite/client/plugins/fps/FpsDrawListener.java

```diff
@@ -90,7 +86,7 @@ void onFocusChanged(FocusChanged event)
 	@Override
 	public void accept(BufferedImage bufferedImage)
 	{
-		if (alwaysEnabled || (unfocusedEnabled && !isFocused))
+		if (config.alwaysLimitFps() || (config.unfocusedLimitFps() && !isFocused))
```

Comment:
Invert this condition (to prevent having giant if blocks in size of the entire method)